packages/app/package.json

```diff
@@ -111,6 +110,7 @@
     "react-router-dom": "^5.2.0",
     "react-scrollbars-custom": "^4.0.25",
     "react-split-pane": "^0.1.92",
+    "react-use": "^17.2.4",
```

Comment:
I replaced the useInterval library with a general hook library [react-use](https://github.com/streamich/react-use), which also includes the `useMounted` hook facilitated in this PR.